### PR TITLE
Add presigned head url to scan object

### PR DESCRIPTION
**Pull request by EndPositive**
## Description
This PR, if applied, adds `findingHeadLink` and `rawResultHeadLink` to the scan's status. These links are useful in optimization of external services (or hooks) querying the file. It is especially useful when you would like to use something like a [streaming response](https://docs.djangoproject.com/en/3.2/ref/request-response/#django.http.StreamingHttpResponse) while retaining original headers (obtained through head link).

### Checklist

* [x] Test your changes as thoroughly as possible before you commit them. Preferably, automate your test by unit/integration tests.
* [x] Make sure `npm test` runs for the whole project.
* [x] Make codeclimate checks happy

### CODE REVIEW

File: operator/apis/execution/v1/scan_types.go [hunk 1]
```diff
@@ -81,6 +81,11 @@ type ScanStatus struct {
 	// RawResultDownloadLink link to download the raw result file from. Valid for 7 days
 	RawResultDownloadLink string `json:"rawResultDownloadLink,omitempty"`
 
+	// FindingHeadLink link to send HEAD request to the finding json file. Valid for 7 days
+	FindingHeadLink string `json:"findingHeadLink,omitempty"`
+	// RawResultHeadLink link to send HEAD request to raw result file from. Valid for 7 days
```
Comment (J12934) [inline operator/apis/execution/v1/scan_types.go:86]:
```suggestion
	// RawResultHeadLink link to send HEAD request to raw result file. Valid for 7 days
```

**Review by J12934 — COMMENTED** [review]
Okay interesting never had to stream files from S3 (yet). 😀
Nice to have the option now 👍

Just a small copy paste error in the description to make the descriptions match properly.

**Comment by EndPositive** [comment]
One day you will and you'll think of me! Thanks for suggestion :smile: .

**Comment by J12934** [comment]
CRD's need to be regenerated, they still have the typos in them 😉

**Pull request merged**